src/replication/adapter/native/image_registry.go

```diff
@@ -63,24 +62,21 @@ func (n native) FetchImages(filters []*model.Filter) ([]*model.Resource, error)
 		})
 	}
 
+	// replication execution should be failed if no any image found,
+	// and show the reason to user.
+	if resources == nil {
+		return nil, fmt.Errorf("Not find any image of name filter: %s, tag filter: %s", nameFilterPattern, tagFilterPattern)
```

Comment:
Shouldn't return error here, if no resources returned, the execution will skip.

Reply:
But the execution status will become success if nothing to do. User maybe not know reason that's execution is success but task list is empty.

Reply:
The status text will be filled with message : there is no resources to replicate.